react/features/conference/middleware.js

```diff
@@ -1,15 +1,28 @@
 // @flow
 
+import { appNavigate } from '../app';
 import {
     CONFERENCE_JOINED,
-    VIDEO_QUALITY_LEVELS,
-    setReceiveVideoQuality
+    conferenceFailed,
+    KICKED_OUT,
+    setReceiveVideoQuality,
+    VIDEO_QUALITY_LEVELS
 } from '../base/conference';
 import { SET_REDUCED_UI } from '../base/responsive-ui';
 import { MiddlewareRegistry } from '../base/redux';
 import { setFilmstripEnabled } from '../filmstrip';
 import { setToolboxEnabled } from '../toolbox';
 
+/**
+ * FIXME Usually the value from lib-jitsi-meet should be used
```

Comment:
Can't we fix this before this PR lands? Otherwise we'll have 2 things to fix later on.

Reply:
We can. I wanted to ask if you think it's fine to just rename the constant which will break the events API for any app using lib-jitsi-meet.

Reply:
I'd say yes. We can PSA the mailing list if you'd like.

Reply:
Actually I'm stupid it should not be a problem if someone is using a constant JitsiConferenceEvents.KICKED. It should update with the lib-jitsi-meet update.